src/utils/functions.js

```diff
@@ -101,3 +110,50 @@ async function createPagedEmbed(
 		}
 	})
 }
+
+async function createGuildConfig(guild, client) {
+	const owner = guild.owner || (await client.users.fetch(guild.ownerID))
+	// create initial config only if it doesn't exist yet
+	ConfigModel.findOne({ guildId: guild.id }).then((config) => {
+		if (config) return
+		console.log(`Creating config for guild with ID ${guild.id}`)
+		ConfigModel.create({
+			communityname: guild.name,
+			guildId: guild.id,
+			contact: owner.id,
+			apikey: "",
+		})
+	})
+	let embed = new MessageEmbed()
+		.setTitle("Welcome to FAGC")
+		.setColor(client.config.embeds.color)
+		.setFooter(client.config.embeds.footer)
+		.setTimestamp()
+	embed.addFields(
+		{ name: "FAGC Invite", value: client.config.fagcInvite },
+		{
+			name: "Initial Setup",
+			value:
+				"We assume that you want to set your guild up. For now, your guild data has been set to a few defaults, " +
+				"such as the guild contact being the guild's owner. To change this, you can always run `fagc!setup`. " +
+				"Run `fagc!help` to view command help. Commands don't work in DMs!",
```

Comment:
Message neeeds updating now that setup has been removed. It should include how everything is readonly until setApiKey is used.